src/components/SearchForm.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Field, Form, withFormik } from 'formik';
+import * as Yup from 'yup';
+
+const SearchForm = ({
+  values,
+  errors,
+  touched,
+}) => (
+  <Form>
+    <div className="form-field">
+      <label>Include these words or phrases
+        <Field
+          type="text"
+          name="searchTerms"
+          placeholder="seperate search terms with a comma (,)"
+        />
+      </label>
+      {touched.searchTerms && errors.searchTerms &&
+      <p className="form-error">{errors.searchTerms}</p>}
+    </div>
+    <div className="form-field">
+      <label>Location (optional, zipcode)
+        <Field
+          type="text"
+          name="location"
+          placeholder="97015"
```

Comment:
I'd avoid putting a valid zip code as a placeholder in case the user actually is from that zip code and thinks the field is already filled out.